### ISSUE TITLE
feature request - support rxjs v6 and angular 6

### ISSUE THREAD

**Issue by sfabriece** [scttcper/ngx-toastr#422]
Support for angular v6 and rxjs v6

**Comment by Boboss74** [comment]
In the readme you can update the path and the file name in this part

If you are using angular-cli you can add it to your angular.json
```json
"styles": [
  "styles.scss",
  "node_modules/ngx-toastr/toastr.css"
]
```

**Comment by yarrgh** [comment]
Yeah, the peerDependencies should be something like "^5.0.0||^6.0.0" to be correct semver. 

I'll open a PR as soon as I'm able, unless someone beats me to it.

**Comment by yarrgh** [comment]
@andelz this should now be fixed in 8.5.1.

I've just ran through a brand new angular-cli app (1.7.4), installed ngx-toastr 8.5.1 and ran through the upgrade process and don't get an error during `ng update @angular/core`. 

Try your project again and let us know if you have any more issues.

**Comment by yarrgh** [comment]
@Boboss74 Yeah, that'll need to be updated. I'll open another issue to track it.
